swarmsim/engine.nim

```diff
@@ -0,0 +1,6 @@
+import engine/protocol
+import engine/network
+import engine/schedulableevent
+import timeutils
+
+export protocol, network, schedulableevent, timeutils
```

Comment:
Doing this makes it easier to use `engine` stuff in other modules. Plus it gives you chance to control which export you might want "users" to see.